### DISCUSSION TITLE
Discord Loot Logger - Moderator tools?

### DISCUSSION THREAD

**Discussion by AJLovelock** [runelite/runelite#16712] [Q&A] [answered]
We have had to remove someone from our discord server due to them breaking our rules. However, even when removed and banned from our discord server. The Discord Loot Logger plugin still displays there drops on the discord server. Is there a way to remove that user from displaying? Or do they have to physically remove the webhook themselves?

**Comment by Felanbird** [comment, marked as answer]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.


But I'll go ahead and answer your question which is no. The way discord webhooks work means if someone you don't want, gets it, you'll need to generate a new link. It has nothing to do with any Runelite plugin.
